arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedColumnIterator.java

```diff
@@ -193,16 +193,6 @@ protected int nextBatchOf(
     }
   }
 
-  public class FixedWidthTypeBinaryBatchReader extends BatchReader {
```

Comment:
Maybe I am daft but it looks like you removed fixed width readers but I don't see where you added any readers?